examples/policy_generate_networkPolicy.yaml

```diff
@@ -1,28 +1,32 @@
 apiVersion: kyverno.io/v1alpha1
 kind: Policy
 metadata:
-  name: "deny-ingress-traffic"
+  name: "default-networkPolicy"
 spec:
   rules:
-  - name: "deny-ingress-traffic"
+  - name: "default-networkPolicy"
     match:
       resources: 
         kinds:
         - Namespace
         name: "devtest"
     generate: 
       kind: NetworkPolicy
-      name: deny-ingress-traffic
+      name: default-networkPolicy
       data:
         spec:
+          # select all pods in the namespace
           podSelector: {}
           policyTypes: 
           - Ingress
+          - Egress
           ingress:
           - from:
-            - podSelector:
+            # allow traffic from any pod in a 
+            # namespace with the label app=prod
+            - namespaceSelector:
                 matchLabels:
-                  role: frontend
-        metadata:
-          labels:
-            policyname: "default"
\ No newline at end of file
+                  app: prod
```

Comment:
why do we need a namespace selector here?

Reply:
if you leave the namespace selector out, it applies to all pods in the current namespace by default